apps/web/src/server/app/validation/validation.route.js

```diff
@@ -27,4 +28,7 @@ router.route(`/${routes.validAppealOutcome.path}`).get(appealDataGuard, getValid
 router.route(`/${routes.invalidAppealOutcome.path}`).get(appealDataGuard, getInvalidAppealOutcome);
 router.route(`/${routes.incompleteAppealOutcome.path}`).get(appealDataGuard, getIncompleteAppealOutcome);
 
+router.route('/appealOutcomeInvalidReason')
```

Comment:
I defined the routes for this already. Please remove this one and use the existing ones. Thank and sorry for the back and forth.